src/components/Project/index.js

```diff
@@ -284,45 +112,31 @@ export default class Project extends React.Component {
   }
 
   render() {
-    const { loading, project, variables, errors, error } = this.props.data;
-    const { store, socket, user } = this.props;
-
-    if(loading){
-      return (
-        <div>
-          Loading ...
-        </div>
-      )
-    }
-
-    if(project)
-      this.props.store.app.setProjectTitle(project.slug)
-
-    console.log(this.props)
+    const { store, socket, user, match, envId, history } = this.props;
 
     return (
       <div className={styles.root}>
         <Switch>
           <Route exact path='/projects/:slug' render={(props) => (
-            <ProjectFeatures project={project} store={store} {...this.props} />
+            <ProjectFeatures match={match} envId={envId} />
```

Comment:
Does it make more sense to have envId in the store and not pass it as prop?

Reply:
yup, changed